src/main/java/org/terasology/launcher/TerasologyLauncher.java

```diff
@@ -131,6 +132,8 @@ public void handle(final WorkerStateEvent workerStateEvent) {
             }
         });
 
+        launcherInitTask.setOnFailed(event -> {openCrashReporterAndExit(new RuntimeException("FUUU"));});
```

Comment:
`"FUUU"` - on point, but I guess we can improve that 😅

Reply:
Oh, forget too. 
Ok, will improve.